### DISCUSSION TITLE
Recover data from db.sqlite3

### DISCUSSION THREAD

**Discussion by cfrontin** [babybuddy/babybuddy#910] [Support]
I had an instance of babybuddy just short of three years ago and only have a zipfile of some of the old build in a backup drive.

Now, on a new server, I have a new instance set up for a new kid. The only way I can see to recover the old kid's data is via the db.sqlite3 file in the old data. I'd like to have it in the new instance/setup. Is there a way to extract or migrate data from the sqlite data for automatically migrating just one kid and all their old observations?

**Comment by cdubz** [comment]
Hm. If you already have the new instance with _new_ data, what you could probably do is set up a temporary instance with the _old_ database and use the [import/export functionality](https://docs.baby-buddy.net/import-export/) to take from the old and add to the new.

If you don't have any new data in your new instance yet, in theory you could just plop the old database in there, run migrations, and have all the old data + a ready state to add new data in the latest version.

**Reply by cfrontin** [reply]
I’d tried running migrations on it but it led to a 500 internal server error after the login screen on the new instance! I then hit the panic button and started from scratch lol.

i got the same error after trying to unpack, reinstall, and update the original server too. I’ll see if I can get it fired up again on the backup instance and go from there
